### DISCUSSION TITLE
Implementing Feature flags using azure | Beneath Abstraction

### DISCUSSION THREAD

**Discussion by giscus** [gopkumr/gopkumr.github.io#4] [General]
# Implementing Feature flags using azure | Beneath Abstraction

Feature flag is a very popular practice in modern application development, which is used to specifically hide features implemented that are not yet ready to be used by wider audience, and when ready can be enabled by a flip of a switch. The flags can also be used as a kill switch for application feature when it not working as expected.
With feature flags implemented, it would be effective to have the features enabled or disabled from a location outside of the application infrastructure or configuration, this way we can have features spanning across applications be controlled via a centralized flag.

http://localhost:1313/post/azurefeatureflags/

**Comment by gopkumr** [comment]
This is a test comment
